src/main/java/ServerPort.java

```diff
@@ -0,0 +1,37 @@
+package com.td.HTTPServer;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.Date;
+
+public class ServerPort {
```

Comment:
yeah, I wasn't very confident with that name myself. But was struggling with a better name.